assets/js/autocomplete/suggestions.js

```diff
@@ -199,7 +199,33 @@ function findMatchingChildren (elements, parentId, term, key) {
 }
 
 /**
- * How good th
+ * Chooses the right sorting priority for the given item.
+ *
+ * @param {Object} item Search result item.
+ *
+ * @returns {number} Sorting priority for a given item. Higher priority means higher position in search results.
+ */
+function getSortingPriority (item) {
+  if (isException(item)) {
+    return sortingPriority['Exception']
+  }
```

Comment:
I am thinking we should remove this and not special case Exceptions in any way. Thoughts?